cloudsim-plus/src/main/java/org/cloudbus/cloudsim/power/PowerMeasurement.java

```diff
@@ -0,0 +1,50 @@
+package org.cloudbus.cloudsim.power;
+
+import org.cloudbus.cloudsim.power.models.PowerModel;
+
+/**
+ * Measurement produced by a {@link PowerModel}.
+ *
+ * A PowerMeasurement consists of a static and a dynamic fraction.
+ */
+public class PowerMeasurement {
```

Comment:
I understand that power consumption is static + dynamic usage.
However, the static usage is supposed to be constant all the time.
I don't know any work which confirms it. But considering that, 
the `staticUsage` value for different instances of `PowerMeasurement`
for the same Host will always be the same.
This way, I don't see the need for a new class.
The `PowerModelSimple` already has a `getStaticPower()` method.

Reply:
`staticUsage` is constant as long as the entity is powered on, but if you shut it off it will drop to 0 (or some very small value). So it's not always the same.

Another example is DVFS: This technique reduces the voltage of a CPU when it has a low load, which also reduces its static power usage.

In other words, every time you try to save energy you usually tackle the static part, so it makes a lot of sense to have this data available in the analysis.